source/making_payments/index.html.md.erb

```diff
@@ -38,8 +38,17 @@ For example:
 
 The amount must be a number data type rather than a string.
 
-The minimum amount is one pence. The maximum amount is 10,000,000 pence (£100,000). [Contact us](/support_contact_and_more_information/#contact-us) if you need to take payments above the maximum amount.
+The minimum amount is one pence plus your Payment Service Provider's (PSP's) transaction fee. To see your PSP's transaction fee, check your contract by either:
 
+- signing into the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) if you use Stripe
+- asking your PSP
+
+The maximum amount is either:
+
+- 10,000,000 pence (£100,000)
+- 5,000,000 pence (£50,000) if you use the [Government Banking Service](https://www.gov.uk/government/groups/government-banking-service-gbs) 
```

Comment:
So, Pay can handle payments of up to £100k. However, you may have a lower maximum payment set by your PSP. The default maximum for Gov Banking services is £5k. If you want to change that maximum, contact Gov Banking. 

If you want to take payments above Pay's maximum, contact us.

I think Gov Banking have ditched 'service' when talking about themselves.

Reply:
Thanks @miriamalice - updated!